cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	// future enabled flags
+	ServiceCatalog      = createFlag("service-catalog", SetBool, nil, nil, true)
+	ExtraClusterUpFlags = createFlag("extra-clusterup-flags", SetString, nil, nil, true)
```

Comment:
-1
I consider the suggested environment variable approach first very unclean. It means they are not explicitly set on the start command.